src/geophires_x/CylindricalReservoir.py

```diff
@@ -56,6 +57,7 @@ def __init__(self, model: Model):
 
         self.OutputDepth = self.ParameterDict[self.OutputDepth.Name] = floatParameter(
             "Cylindrical Reservoir Output Depth",
+            value=self.InputDepth.value,
```

Comment:
Remove - only `DefaultValue` should be specified, not `value` (which is redundant with `DefaultValue` in this case anyway)